rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json

```diff
@@ -860,29 +859,229 @@
         }
       }
     },
-    "cover_art": {
-      "title": "Cover art",
-      "description": "Vendor cover art URL.",
-      "type": "string",
-      "format": "uri"
-    },
-    "electronic_location": {
+    "electronicLocator": {
       "title": "Electronic Locations",
       "description": "Information needed to locate and access an electronic resource.",
       "type": "array",
       "minItems": 1,
       "items": {
         "type": "object",
-        "title": "Electronic Locations",
+        "title": "Electronic Location",
         "required": [
-          "uri"
+          "type",
+          "url"
+        ],
+        "propertiesOrder": [
+          "type",
+          "url",
+          "content",
+          "publicNote"
         ],
         "properties": {
-          "uri": {
-            "title": "Uniform Resource Identifier",
-            "description": "Uniform Resource Identifier (URI), which provides standard syntax for locating an object using existing Internet protocols.",
+          "url": {
+            "title": "url",
+            "description": "Record a unique URL here.",
```

Comment:
Is this description comes form the PO. Seems weird. Same for other descriptions.

Reply:
Its coming from the PO